src/index.ts

```diff
@@ -326,6 +330,28 @@ export async function install(
     };
     const packageBundle = await rollup.rollup(inputOptions);
     await packageBundle.write(outputOptions);
+
+    if (include && legacy) {
+      function renameImportPlugin() {
+        return {
+          name: 'rename-import-plugin',
+          resolveId(source) {
+            if (source.startsWith('/web_modules/')) {
+              return {
+                id: source.replace(/^\/web_modules\//, path.join(cwd, 'node_modules', path.sep)),
+              };
+            }
+            return null;
+          },
+        };
+      }
```

Comment:
I tried to find a Rollup plugin to rename imports, but I couldn’t find one (I don’t think the node resolver works like this).

This is a made-up one that should probably live in a better place (maybe even its own repo).

Anyway simply removing `/web_modules/` didn’t seem to work; Rollup seems to need to replace it with `/node_modules/`.

Reply:
What if we leave the IDs alone, and bundle with the Snowpack-created `web_modules/` files instead of the files inside of node_modules? It would be nice to guarantee the same code being used in both places to minimize legacy-only bugs.

Reply:
Oh smart! I didn’t think of that.